Part5

pomozoff opened the pull request:
Just made some refactoring.

Commit:
[Mod]: Changed UITableView to visible one and added constaraints

Commit:
[Ref]:
- changed var to let
- renamed variables
- made code more readable

Commit:
[Ref]:
- changed var to let
- made code more readable
- use NSURLSession instead NSURLConnection

jquave commented:
Pomozoff, thank you for contributing but many of these changes are present by the end of the tutorial, aren't they?

Commit:
[Add]: Xcode schemes

pomozoff commented:
@jquave, nope, next parts just add new functionality. I have slightly refactored your code to make it more readable, changed some var's with let's and replaced NSURLConnection with NSURLSessions. Moreover I've fixed classical issue with placing just loaded old thumbnail images to the dequeued cell with new content which is loading new thumbnail for this content.
